### PR TITLE
25 create the main section of sprite section page 2

### PR DESCRIPTION
**Pull request by jasonzhwang**
image

AC1: create sprint dash circular page as top
AC2: create sprint summary page as bottom
AC3: click any 'Right' or 'Wrong' button in the sprint dash page, which can redirect to the sprint summary page.

new shared components: ScrollArea CircularDashboard Label

### CODE REVIEW

**Review by tatoMa — COMMENTED** [review]
image

**Comment by tatoMa** [comment]
image

File: src/components/SprintSummary/SprintSummary.module.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      left: -10.44px;
+      top: -5%;
+      bottom: 6.25%;
+    }
+    p {
+      font-size: 16px;
+      font-weight: 700;
+      margin: 0 30px 0 10.44px;
+      margin-top: 0;
+    }
```
Comment (Double-Twelve) [inline src/components/SprintSummary/SprintSummary.module.scss:65]: not the same style as figma

File: src/components/Shared/ScrollArea/ScrollArea.module.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        display: flex;
+        align-items: center;
+        p:first-of-type {
+          font-weight: 600;
+          font-size: 16px;
+        }
+        p {
+          padding: 0 0;
+          margin: 0 0;
+        }
```
Comment (Double-Twelve) [inline src/components/Shared/ScrollArea/ScrollArea.module.scss]: not the same style as figma

File: src/components/Shared/CircularDashboard/CircularDash.module.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  position: relative;
+  .dashInfoContainer {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```
[the remark this answers is not in the document]
Comment (jasonzhwang) [reply src/components/Shared/CircularDashboard/CircularDash.module.scss]: Hi Double-Twelve,

Some style issues different from figma design you mentioned have been fixed.
1.strokeWidth set to be 8 for sprintdash component.
2.use relative top function and use left50% top 50%, and translate(-50%,-50%) to make the dash info always stand in the centre of Circlular dash.
3.remove unnecessary <div> and <> to wrap JSX element for DashInfo.
4.zIndex for DashInfo is removed as I forgot to remove while refactoring.
5.box-shadow has been added as figma.
6.Missing p tag style has been styled as figma design in sprint summary and scroll area.

Thanks.

**Review by LULIU-9 — APPROVED** [review]
LGTM

**Pull request merged**